packages/design-system-tailwind-preset/src/typography.test.ts

```diff
@@ -11,14 +11,6 @@ describe('Typography', () => {
 
```

Comment:
Moved to separate PR https://github.com/MetaMask/metamask-design-system/pull/699